app/views/publishers/vacancies/edit_vacancy_sections/_edit_applying_for_the_job.html.slim

```diff
@@ -1,4 +1,4 @@
-= render "shared/error_tag", attributes: %i[contact_email contact_number school_visits how_to_apply application_link], model: @vacancy
+= render "shared/error_tag", applying_for_the_job_fields, model: @vacancy
```

Comment:
This does not seem consistent with how it is used on the other partials in this commit. I would have expected it to be:
`= render "shared/error_tag", attributes: applying_for_the_job_fields, model: @vacancy`
seems there is a missing `attributes:`